Basic/Complex/complex.pd

```diff
@@ -25,6 +25,34 @@ pp_addpm {At => Top}, <<'EOD';
 
 PDL::Complex - handle complex numbers
 
+=head2 Native complex support
+
+There are important changes in complex number handling. Native C level support
+has been added, most functions and modules in the core distribution are supporting
+these as well. 
+
+PDL can now handle complex numbers natively as scalars. This has the advantage that
+real and complex valued piddles have the same dimensions. Consider this when writing
+code in the future.
+
+Important functions are:
+
+=for
```

Comment:
Are you sure this is the right markup? Is it used somewhere else, or should this actually be `=over`?